apps/frontend/src/network/teams/__tests__/About.test.tsx

```diff
@@ -116,4 +117,22 @@ describe('the team groups card', () => {
       expect(queryByText(/team groups/i)).toBeInTheDocument(),
     );
   });
+
+  it('renders link for group', async () => {
+    const { queryByText, getByText } = await renderTeamAbout(
+      {
+        team: createTeamResponse(),
+      },
+      {
+        items: [{ ...createGroupResponse(), name: 'example group', id: 'g1' }],
+        total: 1,
+      },
+    );
+    await waitFor(() =>
+      expect(queryByText(/team groups/i)).toBeInTheDocument(),
+    );
+    expect(
+      getByText(/example group/i, { selector: 'h4' }).parentElement,
+    ).toHaveAttribute('href', '/network/groups/g1');
```

Comment:
```suggestion
    ).toHaveAttribute('href', expect.stringMatching(/g1$/));
```
don't want a hundred tests breaking if we change the route structure